docs/generate_example_images.py

```diff
@@ -8,62 +9,72 @@
 
 import kornia as K
 
-mod = importlib.import_module("kornia.augmentation")
 
-BASE_IMAGE_URL: str = "https://raw.githubusercontent.com/kornia/data/main/panda.jpg"
-OUTPUT_PATH = Path(__file__).absolute().parent / "build/html/_static"
-os.makedirs(OUTPUT_PATH, exist_ok=True)
+def main():
 
-# perform request
-response = requests.get(BASE_IMAGE_URL).content
+    mod = importlib.import_module("kornia.augmentation")
 
-# convert to array of ints
-nparr = np.frombuffer(response, np.uint8)
+    BASE_IMAGE_URL: str = "https://raw.githubusercontent.com/kornia/data/main/panda.jpg"
+    OUTPUT_PATH = Path(__file__).absolute().parent / "source/_static/img"
+    os.makedirs(OUTPUT_PATH, exist_ok=True)
+    print(f"Pointing images to path {OUTPUT_PATH}.")
 
-# convert to image array
-img: np.ndarray = cv2.imdecode(nparr, cv2.IMREAD_UNCHANGED)
+    # perform request
+    response = requests.get(BASE_IMAGE_URL).content
 
-# convert the image to a tensor
-img_t: torch.Tensor = K.utils.image_to_tensor(img, keepdim=False)  # 1xCxHXW
-img_t = img_t.float() / 255.0
+    # convert to array of ints
+    nparr = np.frombuffer(response, np.uint8)
 
-# TODO: make this more generic for modules out of kornia.augmentation
-# Dictionary containing the transforms to generate the sample images:
-# Key: Name of the transform class.
-# Value: The default parameters to use.
-augmentations_list: dict = {
-    "ColorJitter": (0.1, 0.1, 0.1, 0.1),
-    "RandomAffine": ((-15.0, 20.0),),
-    "RandomBoxBlur": ((7, 7),),
-    "RandomChannelShuffle": (),
-    "RandomErasing": ((0.4, 0.8), (0.3, 1 / 0.3)),
-    "RandomElasticTransform": ((63, 63), (32, 32), (1.0, 1.0)),
-    "RandomEqualize": (),
-    "RandomFisheye": (torch.tensor([-0.3, 0.3]), torch.tensor([-0.3, 0.3]), torch.tensor([0.9, 1.0])),
-    "RandomGrayscale": (),
-    "RandomGaussianNoise": (0.0, 0.05),
-    "RandomHorizontalFlip": (),
-    "RandomInvert": (),
-    "RandomMotionBlur": (7, 35.0, 0.5),
-    "RandomPerspective": (0.2,),
-    "RandomPosterize": (3,),
-    "RandomResizedCrop": ((510, 1020), (3.0, 3.0), (2.0, 2.0)),
-    "RandomRotation": (45.0,),
-    "RandomSharpness": (1.0,),
-    "RandomSolarize": (0.1,),
-    "RandomVerticalFlip": (),
-    "RandomThinPlateSpline": (),
-}
+    # convert to image array
+    img: np.ndarray = cv2.imdecode(nparr, cv2.IMREAD_UNCHANGED)
 
-# ITERATE OVER THE TRANSFORMS
+    # convert the image to a tensor
+    img_t: torch.Tensor = K.utils.image_to_tensor(img, keepdim=False)  # 1xCxHXW
```

Comment:
A quick note to resize the image first. Otherwise, it gonna be too heavy for webpages.